pages/board/index.tsx

```diff
@@ -65,8 +65,10 @@ export default function Board() {
       <section className={styles["best-section"]}>
         <h2>베스트 게시글</h2>
         <div className={styles["bestpost-container"]}>
-          {BestArticles.map((article) => (
-            <BestPost article={article} key={article.id} />
+          {bestArticles.map((article) => (
+            <Link href={`/${article.id}`}>
```

Comment:
경로에 board까지 포함되어야 정상적으로 동작할 것 같네요.

```suggestion
            <Link href={`/board/${article.id}`}>
```